assets/scripts/gallery/view.js

```diff
@@ -1,18 +1,19 @@
 /* global app, system, street, galleryUserId, signedIn, signInData, mode, abortEverything */
 /* global MODES, ERRORS, URL_NEW_STREET, URL_NEW_STREET_COPY_LAST, DEFAULT_NAME */
 /* global _sendDeleteStreetToServer, _sendDeleteStreetToServer, _hideControls,
-      _onWindowFocus, _updatePageUrl, _updateToLatestSchemaVersion, _getStreetUrl,
-      _updateScrollButtons */
+      _updatePageUrl, _updateToLatestSchemaVersion, _getStreetUrl */
 import { trackEvent } from '../app/event_tracking'
 import { showError } from '../app/errors'
 import { msg } from '../app/messages'
+import { onWindowFocus } from '../app/focus'
 import { formatDate } from '../util/date_format'
 import { removeElFromDOM } from '../util/dom_helpers'
 import { fetchGalleryData } from './fetch_data'
 import { fetchGalleryStreet } from './fetch_street'
 import { drawStreetThumbnail } from './thumbnail'
 import { hideStatusMessage } from '../app/status_message'
 import { StreetName } from '../streets/name_sign'
+import { updateScrollButtons } from '../gallery/scroll'
```

Comment:
There is no need to import from `../gallery/scroll`, You can import from `./scroll` here.